lib/ansiblelint/formatters/__init__.py

```diff
@@ -87,6 +87,26 @@ def format(self, match, colored=False):
                                     match.message)
 
 
+class AnnotationsFormatter(BaseFormatter):
+    """Emits errors in format that is recognized by Github Annotation parsing.
+
+    ::warning file={name},line={line},col={col}::{message}
+    """
+
+    def format(self, match, colored=False):
```

Comment:
This should probably print both to console and to workflow commands. Because the log must still remain readable.